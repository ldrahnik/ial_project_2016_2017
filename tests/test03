A 1 B A -1 C 2 B
