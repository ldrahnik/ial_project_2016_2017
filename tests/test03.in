A 1 B 
A -1 C
C 2 B
