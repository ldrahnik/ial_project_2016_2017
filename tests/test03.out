FloydWarshall: F -(2)> D -(6)> B -(7)> C -(11)> A 
BellmanFord: F -(2)> D -(6)> B -(7)> C -(11)> A
